doc/source/io.rst

```diff
@@ -1994,6 +1994,18 @@ into a flat table.
 .. ipython:: python
 
    from pandas.io.json import json_normalize
+   data = [
+             {'id': 1, 'name': {'first': 'Coleen', 'last': 'Volk'}},
+             {'name': {'given': 'Mose', 'family': 'Regner'}},
+             {'id': 2, 'name': 'Faye Raker'}
+          ]
+   json_normalize(data)
+
+.. _io.jsonl:
+
+.. ipython:: python
+
+   from pandas.io.json import json_normalize
```

Comment:
You should be able to add this to the same same `..ipython:: python` block. Anytime there's a line that produces output / isn't assigned, it gets printed out. So like

```
data  = [ # your new example ]
json_normalize(data)

data = [# old example]
json_normalize(data)